iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceMethodCommon.java

```diff
@@ -255,36 +254,19 @@ public static void classSetup()
         }
     }
 
-    public void cleanToStart() throws Exception
+    public void openDeviceClientAndSubscribeToMethods() throws Exception
     {
         testInstance.setup();
-        actualStatusUpdates = new ArrayList<Pair<IotHubConnectionStatus, Throwable>>();
-        setConnectionStatusCallBack(actualStatusUpdates);
+        testInstance.deviceTestManager.client.open();
 
         try
         {
-            this.testInstance.deviceTestManager.tearDown();
-        }
-        catch (IOException e)
-        {
-            e.printStackTrace();
-        }
-        catch (InterruptedException e)
-        {
-            e.printStackTrace();
-        }
-
-        this.testInstance.deviceTestManager.clearDevice();
-
-        try
-        {
-            this.testInstance.deviceTestManager.setup(true, false);
-            IotHubServicesCommon.confirmOpenStabilized(actualStatusUpdates, 120000, this.testInstance.deviceTestManager.client);
```

Comment:
All of the above logic isn't necessary since the test client starts off closed, are opened elsewhere, and we no longer have connection stability issues that require us to use this ```confirmOpenStabilized(...)``` anymore.

Reply:
As a backstory for this ```confirmOpenStabilized()```, our AMQP layer used to be unstable and our tests would have to wait a bit for the instability to go away before continuing. I fixed that instability a while back, so we really don't need this anymore